test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -437,22 +440,32 @@ public void HubsCannotHaveOverloadedMethods()
             }
         }
 
-        [Fact]
-        public async Task BroadcastHubMethod_SendsToAllClients()
+        [Theory]
+        [InlineData("dynamic")]
+        [InlineData("default")]
+        public async Task BroadcastHubMethodSendsToAllClients(string hubType)
         {
             var serviceProvider = CreateServiceProvider();
 
-            var endPoint = serviceProvider.GetService<HubEndPoint<MethodHub>>();
+            dynamic endPoint;
+            if (hubType.Equals("dynamic"))
```

Comment:
You should not need to do that - InlineData should contain the type (i.e. [InlineData(typeof(DynamicType)]) and then you would use reflection to create the type.